content/docs/specs/native_histograms.md

```diff
@@ -0,0 +1,2353 @@
+never be represented explicitly. The only cost is the additional zero element
+at the beginning of the custom values list.)
+
+The last custom value MUST not be `+Inf`. Observations greater than the last
+custom value go into an overflow bucket with an upper boundary of `+Inf`. This
+overflow bucket is added with an index equal to the length of the custom
+values list.
+
+### Exemplars
+
+A native histogram sample can have zero, one, or more exemplars. They work in
+the same way as conventional exemplars, but they are organized in a list (as
+there can be more than one), and they MUST have a timestamp.
+
+Exemplars exposed as part of a classic histogram MAY be used by native
+histograms, if they have a timestamp.
+
+### Special cases of observed values
+
+Instrumented code SHOULD avoid observing values of `NaN` and `±Inf` because
+they make limited sense in the context of a histogram. However, those values
+MUST still be handled properly, as described in the following.
+
+The sum of observations is calculated as usual by adding the observation to the
+sum of observations, following normal floating point arithmetic. (For example,
+an observation of `NaN` will set the sum to `NaN`. An observation of `+Inf`
+will set the sum to `+Inf`, unless it is already `NaN` or `-Inf`, in which case
+the sum is set to `NaN`.)
+
+An observation of `NaN` goes into no bucket, but increments the count of
+observations. This implies that the count of observations can be greater than
+the sum of all buckets, and the difference is the number of `NaN` observations.
+
+An observation of `+Inf` or `-Inf` increments the count of observations and
+increments a bucket chosen in the following way:
+- With a standard schema, a `+Inf` observation increments the
+  _positive overflow bucket_ as described above.
+- With a standard schema, a `-Inf` observation increments the
+  _negative overflow bucket_ as described above.
+- With schema -53 (custom buckets), a `+Inf` observation increments the
+  bucket with an index equal to the length of the custom values list.
+- With schema -53 (custom buckets), a `-Inf` observation increments the
+  bucket with index zero.
+
+### OpenTelemetry interoperability
+
+Prometheus (Prom) native histograms with a standard schema can be
+easily mapped into an OpenTelemetry (OTel) exponential histogram and vice
+versa, as detailed in the following.
+
+The Prom _schema_ is equal to the _scale_ in OTel, with the restriction that
+OTel allows lower values than -4 and higher values than +8. As described above,
+Prom has reserved more schema numbers to extend its range, should it ever by
+required in practice.
+
+The index is offset by one, i.e. a Prom bucket with index _n_ has index _n-1_
+for OTel.
+
+OTel has a dense rather than a sparse representation of buckets. One might see
+OTel as “Prom with only one span”.
+
+The Prom _zero bucket_ is called _zero count_ for OTel, but works the same,
```

Comment:
I believe the attribute is called ZeroCount in both the Histogram/Float Histogram struct in Prometheus and the OTel Exponential Histogram